### PR TITLE
add comments for rec

### PR DESCRIPTION
**Pull request by tink2123**

### CODE REVIEW

File: ppocr/utils/character.py [hunk 1]
```diff
@@ -132,6 +151,21 @@ def cal_predicts_accuracy(char_ops,
                           labels,
                           labels_lod,
                           is_remove_duplicate=False):
+    """
+    Calculate predicts accrarcy
```
Comment (littletomatodonkey) [inline ppocr/utils/character.py:155]: prediction

File: tools/program.py [hunk 2]
```diff
@@ -150,19 +150,20 @@ def check_gpu(use_gpu):
...
     """
     Build a program using a model and an optimizer
-        1. create feeds
-        2. create a dataloader
-        3. create a model
-        4. create fetchs
-        5. create an optimizer
+        1. create a dataloader
+        2. create a model
+        3. create fetchs
```
Comment (littletomatodonkey) [inline tools/program.py:155]: fetches

File: ppocr/utils/character.py (hunk 1 above)
Comment (tink2123) [reply ppocr/utils/character.py]: Done, thx

File: tools/program.py (hunk 2 above)
Comment (tink2123) [reply tools/program.py]: Done, thx

File: ppocr/utils/character.py (hunk 1 above)
Comment (tink2123) [reply ppocr/utils/character.py]: done, thx

File: tools/program.py (hunk 2 above)
Comment (tink2123) [reply tools/program.py]: Done, thx

**Pull request merged**